pkg/api/VERSIONING.md

```diff
@@ -95,7 +95,7 @@ number.
 Note: Components v0.6.0 and below did not have a versioned protocol between the controller and the runner.
 | Release | controller | runner |
 |---------|------------|--------|
-| _Current_ | 0 - 1 | 1 |
```

Comment:
Side note: the versioning scheme here is a little bit not intuitive, and I was thinking whether it would simplify things if we would just have "the autoscaling protocol", meaning, would use the same versioning scheme for all components protocols.

Reply:
I'm not sure tbh.

From my perspective, the primary reason to have a versioned protocol is to make releasing & rollback more clear - especially when things are already off the happy path. Having a single versioning scheme would reduce information.

At the same time, I think the information it currently gives us is insufficient and sometimes misleading by omission. For example, the changes from your PR #865 weren't reflected in any versioning here, but _did_ have compatibility implications. I think our lack of existing coverage for more complex relationships like that is a strong reason to switch to a unified versioning scheme.

Could you open an issue for it?

Reply:
Opened https://github.com/neondatabase/autoscaling/issues/888

Reply:
thanks!